package.json

```diff
@@ -27,6 +27,7 @@
     "@types/clipboard": "^2.0.1",
     "@types/pluralize": "0.0.29",
     "babel-eslint": "^10.0.2",
+    "babel-plugin-dynamic-import-amd": "github:holyzfy/babel-plugin-dynamic-import-amd",
```

Comment:
Why are we getting this from a fork instead of `npm`? Also, maybe lock its version to a tag/branch/commit.

Reply:
Because the original npm package is so old and incompatible with Babel 7. RequireJS is old anyway...